.rubocop.yml

```diff
@@ -26,6 +26,21 @@ Style/FrozenStringLiteralComment:
 Layout/EndOfLine:
   EnforcedStyle: lf
 
+Layout/ClassStructure:
+  Enabled: true
+  Categories:
+    include:
+      - prepend
+  ExpectedOrder:
+      - extend
+      - include
+      - constant
+      - public_class_method
```

Comment:
I think all those should be in plural - `class_method_definitions` or `class_methods`.

Reply:
Ok. I’ll make them pluralized. Makes sense.